index.html

```diff
@@ -238,24 +235,25 @@ <h1>Recent Events</h1>
                             <div class="event-header">
                                 <!-- Title -->
                                 <div class="event-title">
-                                    <span class="event-type">Reading Group</span>
-                                    <span class="event-name"><a href="https://arxiv.org/abs/1904.01766">
-                                        VideoBERT: a Joint Model for Video and Language Learning
+                                    <span class="event-type">Tutorial Session</span>
+                                    <span class="event-name"><a href="https://docs.google.com/presentation/d/1jrlcfeAh7MynnFmvvGxQJyQ4rNSS1D3li5bC1Z86mMo/edit#slide=id.p" target="_blank">
```

Comment:
The tutorial repo is more relevant here, and makes it more clear who the original authors were. Let's use this link: https://gitlab.eecs.umich.edu/MSAIL/tensorflow-tutorial

(make sure to update the text below to reflect that too!)